conf/app.ini

```diff
@@ -120,7 +120,7 @@ MINIMUM_KEY_SIZE_CHECK = false
 OFFLINE_MODE = false
 DISABLE_ROUTER_LOG = false
 ; Generate steps:
-; $ ./gogs cert -ca=true -duration=8760h0m0s -host=myhost.example.com
+; $ ./main cert -ca=true -duration=8760h0m0s -host=myhost.example.com
```

Comment:
@andreynering yeah, we will talk about this on https://github.com/go-gitea/docs/issues/3, I will replace here now